contracts/managerRole.sol

```diff
@@ -0,0 +1,133 @@
+//SPDX-License-Identifier: Unlicense
+pragma solidity ^0.8.4;
+
+import "./helpers.sol";
+
+contract managerRole is Helper {
+    constructor(address _instaList, address _instaImplementationM1)
+        Helper(_instaList, _instaImplementationM1)
+    {}
+
+    /**
+     * @dev add a new manager for caller(DSA) along with allowed connectors
+     * @param _manager address to be added as manager
+     * @param _targets array of connector names to be enabled for new manager
+     */
+    function addManagerWithConnectors(
+        address _manager,
+        string[] memory _targets
```

Comment:
Need to add checks that the `_targets` is a valid connector

Reply:
https://github.com/Instadapp/dsa-contracts/blob/master/contracts/v2/registry/connectors.sol#L122